meow/frontend/src/components/EditPost/index.js

```diff
@@ -94,6 +95,10 @@ class EditPost extends React.Component {
     }
   }
 
+  handleMeowWithin15Mins = newWarningState => {
```

Comment:
Nitpick: I would prefer you call this setMeowWithin15Mins (or whatever depending on what you rename the variable) to make it clearer that the function is used to set state